internal/scorecard/alpha/tests/olm.go

```diff
@@ -28,14 +35,51 @@ const (
 )
 
 // BundleValidationTest validates an on-disk bundle
-func BundleValidationTest(bundle registry.Bundle) scapiv1alpha2.ScorecardTestResult {
+func BundleValidationTest(dir string) scapiv1alpha2.ScorecardTestResult {
```

Comment:
BundleValidationTest() will get a bundle passed to it from the test image, so it assumes the signature of the test is BundleValidationTest(bundle registry.Bundle)